src/app/components/header/header.component.scss

```diff
@@ -0,0 +1,62 @@
+@import "src/app/style/vars";
+@import "src/app/style/mixins";
+
+.ng-invalid:not(form) {
+  border-left: 5px solid #a94442; /* red */
```

Comment:
this color is not in our style folder ? If not can not you add it, or use another one ?